### ISSUE TITLE
Test class couldn't be compiled [Bug]

### ISSUE THREAD

**Issue by Pzixel** [PerfDotNet/BenchmarkDotNet#90]
I have following test class:
```csharp
public class OrderByBench
{
    private const int N = 1024*1024*64;
    private readonly Tuple<int, int>[] _data = Enumerable.Range(1, N).Select(x => new Tuple<int, int>(x/10, x%10)).ToArray();

    [Benchmark]
    public Tuple<int, int>[] Linq()
    {
        return Enumerable.OrderBy(_data, tuple => tuple.Item1).ThenBy(t => t.Item2).ToArray();
    }
}
```

and running code:

```csharp
var summary = BenchmarkRunner.Run<OrderByBench>();
```
When I run it I get following output
![default](https://cloud.githubusercontent.com/assets/11201122/12945083/390536a0-cffc-11e5-92b6-f6440b6ffac6.png)
Runned under version 0.9.0

**Comment by AndreyAkinshin** [comment]
Fixed in the `develop` branch.

**Comment by AndreyAkinshin** [comment]
Fixed in `v0.9.1`.
